core/templates/dev/head/pages/community-dashboard-page/question-opportunities/question-opportunities.directive.ts

```diff
@@ -61,15 +101,114 @@ angular.module('oppia').directive('questionOpportunities', [
           };
 
           ctrl.onLoadMoreOpportunities = function() {
-            if (
-              !ctrl.opportunitiesAreLoading &&
+            if (!ctrl.opportunitiesAreLoading &&
                 ctrl.moreOpportunitiesAvailable) {
               ctrl.opportunitiesAreLoading = true;
               ContributionOpportunitiesService.getMoreSkillOpportunities(
                 updateWithNewOpportunities);
             }
           };
 
+          ctrl.onClickButton = function(skillId) {
```

Comment:
Can this name be more descriptive of the button? onClickButton seems too generic.

Reply:
Done.